eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -222,16 +236,21 @@ instance
   ) =>
   ShelleyEraImp (ConwayEra c)
   where
-  initImpNES =
+  initImpNES = do
     let nes =
           initAlonzoImpNES
             & nesEsL . curPParamsEpochStateL . ppDRepActivityL .~ EpochInterval 100
             & nesEsL . curPParamsEpochStateL . ppGovActionLifetimeL .~ EpochInterval 30
+            & nesEsL . curPParamsEpochStateL . ppCommitteeMaxTermLengthL .~ EpochInterval 20
+            & nesEsL . epochStateGovStateL . committeeGovStateL .~ SJust initialCommittee
+        -- TODO: configure sensible voting thresholds, rather than defaulting to zero
```

Comment:
From the looks of it, this PR relaxes a bunch of tests by removing thresholds, which used to be set by a call to `electBasicCommitee`.

I think we need to take care of this TODO in this PR or at the very least make sure we have the same thresholds in tests as before this PR (which probably would mean resetting them to zero in some tests that relied on zero thresholds)